modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/gc/io/GarbageCollectionInnerIo.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.storage.pagememory.mv.gc.io;
+
+import org.apache.ignite.internal.pagememory.io.IoVersions;
+import org.apache.ignite.internal.pagememory.tree.BplusTree;
+import org.apache.ignite.internal.pagememory.tree.io.BplusInnerIo;
+import org.apache.ignite.internal.pagememory.tree.io.BplusIo;
+import org.apache.ignite.internal.storage.pagememory.mv.gc.GarbageCollectionRowVersion;
+import org.apache.ignite.internal.storage.pagememory.mv.gc.GarbageCollectionTree;
+
+/**
+ * IO routines for {@link GarbageCollectionTree} inner pages.
+ */
+public class GarbageCollectionInnerIo extends BplusInnerIo<GarbageCollectionRowVersion> implements GarbageCollectionIo {
+    /** Page IO type. */
+    public static final short T_GARBAGE_COLLECTION_INNER_IO = 15;
```

Comment:
Can we have such constants in separate classes, like in `IndexPageTypes`, for example?

Reply:
Why not.